modules/nf-core/gatk4/genotypegvcfs/main.nf

```diff
@@ -57,7 +57,7 @@ process GATK4_GENOTYPEGVCFS {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     """
-    touch ${prefix}.vcf.gz
+    echo | gzip > ${prefix}.vcf.gz
```

Comment:
Does `echo |` work as fine as `echo "" |`?

Reply:
yeah it does exactly the same thing :)